src/transformers/testing_utils.py

```diff
@@ -986,6 +986,17 @@ def require_torch_gpu(test_case):
     return unittest.skipUnless(torch_device == "cuda", "test requires CUDA")(test_case)
 
 
+def require_torch_large_gpu(test_case, memory: float = 20):
+    """Decorator marking a test that requires a GPU with more than `memory` GiB of memory."""
+    if torch_device != "cuda":
+        return unittest.skip(reason=f"test requires a GPU with more than {memory} GiB of memory")(test_case)
```

Comment:
There is a lot of work on device agnostic. So let's make the message more specific:

> requires a GPU
to
> requires a CUDA GPU